packages/knip/fixtures/subpath-patterns/package.json

```
@@ -1,6 +1,7 @@
 {
   "name": "my-package",
   "imports": {
-    "#internals/*": "./src/internals/*.ts"
+    "#internals/*": "./src/internals/*.ts",
+    "#internals-explicit-ext/*": "./src/internals/*"
```

```suggestion
    "#internals-explicit-ext/*": "./src/internals/*",
    "#internals-alias/used.alt": "./src/internals/used.alt"
```

And that file too then please :)